stripe_app/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from stripe_app.models import Item
+
+
+class ItemAdmin(admin.ModelAdmin):
+    pass
+
+
+admin.site.register(Item, ItemAdmin)
```

Comment:
Так, кстати, моднее)) 
```suggestion
@admin.register(Item)
class ItemAdmin(admin.ModelAdmin):
    pass
```